### ISSUE TITLE
Crash on attempting to choose night theme

### ISSUE THREAD

**Issue by TPS** [florisboard/florisboard#640]
_N.B.: Having perfunctorily searched, I can't tell if this duplicates another bug/crash report._

#### Short description
Crash on attempting to choose night theme

#### Steps to reproduce
1. Go to Theme tab in Floris settings
2. Click on Selected theme under **Night theme**
4. Immediate crash as below

#### Environment information
- FlorisBoard Version:  v0.3.10
- Install Source:  F-Droid
- Device:  Moto G7
- Android version, ROM:  10 stock

```
~~~ 1618269754582.stacktrace ~~~

java.lang.RuntimeException: Unable to start activity ComponentInfo{dev.patrickgold.florisboard/dev.patrickgold.florisboard.settings.ThemeManagerActivity}: android.view.InflateException: Binary XML file line #37 in dev.patrickgold.florisboard:layout/theme_manager_activity: Binary XML file line #37 in dev.patrickgold.florisboard:layout/theme_manager_activity: Error inflating class dev.patrickgold.florisboard.ime.text.keyboard.KeyboardView
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3477)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3620)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2183)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:241)
	at android.app.ActivityThread.main(ActivityThread.java:7604)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:941)
Caused by: android.view.InflateException: Binary XML file line #37 in dev.patrickgold.florisboard:layout/theme_manager_activity: Binary XML file line #37 in dev.patrickgold.florisboard:layout/theme_manager_activity: Error inflating class dev.patrickgold.florisboard.ime.text.keyboard.KeyboardView
Caused by: android.view.InflateException: Binary XML file line #37 in dev.patrickgold.florisboard:layout/theme_manager_activity: Error inflating class dev.patrickgold.florisboard.ime.text.keyboard.KeyboardView
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance0(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
	at android.view.LayoutInflater.createView(LayoutInflater.java:854)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1006)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:961)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1123)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1084)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1126)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1084)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1126)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1084)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1126)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1084)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1126)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1084)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:682)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:534)
	at dev.patrickgold.florisboard.databinding.ThemeManagerActivityBinding.inflate(ThemeManagerActivityBinding.java:101)
	at dev.patrickgold.florisboard.databinding.ThemeManagerActivityBinding.inflate(ThemeManagerActivityBinding.java:95)
	at dev.patrickgold.florisboard.settings.ThemeManagerActivity.onCreateBinding(ThemeManagerActivity.kt:159)
	at dev.patrickgold.florisboard.settings.ThemeManagerActivity.onCreateBinding(ThemeManagerActivity.kt:52)
	at dev.patrickgold.florisboard.ime.core.FlorisActivity.onCreate(FlorisActivity.kt:52)
	at dev.patrickgold.florisboard.settings.ThemeManagerActivity.onCreate(ThemeManagerActivity.kt:124)
	at android.app.Activity.performCreate(Activity.java:7822)
	at android.app.Activity.performCreate(Activity.java:7811)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1328)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3452)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3620)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2183)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:241)
	at android.app.ActivityThread.main(ActivityThread.java:7604)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:941)
Caused by: java.lang.NullPointerException
	at dev.patrickgold.florisboard.ime.core.FlorisBoard$Companion.getInstance(FlorisBoard.kt:182)
	at dev.patrickgold.florisboard.ime.text.gestures.GlideTypingManager$Companion.getInstance(GlideTypingManager.kt:32)
	at dev.patrickgold.florisboard.ime.text.keyboard.KeyboardView.<init>(KeyboardView.kt:80)
	at dev.patrickgold.florisboard.ime.text.keyboard.KeyboardView.<init>(KeyboardView.kt:88)
	... 38 more

```
-->

**Comment by patrickgold** [comment]
Thanks for your bug report! This has already been fixed (see #593), will be released in the next beta release beta01 tomorrow.

**Comment by TPS** [comment]
Thanks very much! 🙇🏾‍♂️
